src/Uno.UI/UI/Xaml/Media/ImageSource.Android.cs

```diff
@@ -213,14 +212,19 @@ private static Bitmap RespectExifOrientation(ExifInterface exifInterface, Bitmap
 
 			var matrix = new Android.Graphics.Matrix();
 			matrix.PostRotate(rotationAngle);
-			bitmap = Bitmap.CreateBitmap(bitmap, x: 0, y: 0, width: bitmap.Width, height: bitmap.Height, matrix, true);
 
-			return bitmap;
+			var createdBitmap = Bitmap.CreateBitmap(bitmap, x: 0, y: 0, width: bitmap.Width, height: bitmap.Height, matrix, true);
+
+			return createdBitmap;
```

Comment:
https://developer.android.com/reference/android/graphics/Bitmap#createBitmap(android.graphics.Bitmap,%20int,%20int,%20int,%20int,%20android.graphics.Matrix,%20boolean)

<html>
<body>
<!--StartFragment--><div class="devsite-table-wrapper" style="box-sizing: inherit; margin: var(--devsite-table-margin,16px 0); padding: 0px; overflow: auto; color: rgb(32, 33, 36); font-family: Roboto, &quot;Noto Sans&quot;, &quot;Noto Sans JP&quot;, &quot;Noto Sans KR&quot;, &quot;Noto Naskh Arabic&quot;, &quot;Noto Sans Thai&quot;, &quot;Noto Sans Hebrew&quot;, &quot;Noto Sans Bengali&quot;, sans-serif; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">

Returns
--
Bitmap | A bitmap that represents the specified subset of source This value cannot be null.

</div><br class="Apple-interchange-newline"><!--EndFragment-->
</body>
</html>Returns
[Bitmap](https://developer.android.com/reference/android/graphics/Bitmap)	A bitmap that represents the specified subset of source This value cannot be null.

Is it incorrectly annotated? Should we report to https://github.com/xamarin/xamarin-android?

Reply:
Thanks for the valuable feedback. This is a valid point 

cc: @jeromelaban